Dockerfile

```diff
@@ -0,0 +1,38 @@
+# syntax=docker/dockerfile:1
+FROM node:21.5
+
+ENV SKIP_MAKE_FILES=${SKIP_MAKE_FILES}
+
+COPY . /Cosmos-Keyboards/
+WORKDIR /Cosmos-Keyboards
+
+RUN apt-get update
+RUN apt-get install -y python3.11 golang gcc libgl1-mesa-dev python3.11-venv python3-pip
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+
+RUN python3 -m venv venv
+RUN ls -la "venv/bin/"
+RUN venv/bin/pip install mkdocs-material[imaging]==9.4.14 mkdocs-awesome-pages-plugin==2.9.2 mkdocs-rss-plugin==1.9.0 lxml==4.9.3
+
+RUN mkdir -p target
+RUN npm install --include=optional
+
+RUN set -a && . ./.env && set +a && \
+    if [ "$SKIP_MAKE_FILES" != "true" ]; then \
+        make || true;\
```

Comment:
I added these thinking there may be a case where make fails we would still want them to be able to build. However, now I'm thinking we probably shouldn't mask or hide errors. 

I am having issues running 'make keyboards'. 
1. Did you install gl globally? I'm not seeing it in dev dependencies. 
2. I'm getting the following error when gl is installed. It seems to be something to do with the way the plate renders. I ran into a similar issue on linux when using WiylderBuild's https://github.com/bullwinkle3000/dactyl-keyboard. I wonder if it's the same problem:

```
file:///Cosmos-Keyboards/src/assets/replicad_single.js:51
                throw ex;
                ^

TypeError: Cannot read properties of undefined (reading '$$')
    at RegisteredPointer.nonConstNoSmartPtrRawPointerToWireType [as toWireType] (file:///Cosmos-Keyboards/src/assets/replicad_single.js:1885:23)
    at BRep_Builder.TopoDS_Builder$Add [as Add] (eval at new_ (file:///Cosmos-Keyboards/src/assets/replicad_single.js:1964:186), <anonymous>:9:26)
    at file:///Cosmos-Keyboards/src/lib/worker/modeling/index.ts:13:21
    at Array.forEach (<anonymous>)
    at combine (file:///Cosmos-Keyboards/src/lib/worker/modeling/index.ts:11:10)
    at Object.plate (file:///Cosmos-Keyboards/src/model_gen/node-model.ts:39:28)
    at generate (file:///Cosmos-Keyboards/src/model_gen/node-model.ts:44:42)
    at async assembleGroup (file:///Cosmos-Keyboards/src/model_gen/node-render.ts:12:37)
    at async render (file:///Cosmos-Keyboards/src/model_gen/node-render.ts:43:35)
    at async renderTrimmed (file:///Cosmos-Keyboards/src/model_gen/keyboards.ts:9:17)

```

Reply:
That's really odd. Cosmos is using a webassembly build of opencascade, so you'd think it would behave the same everywhere. I'll double check that it still runs on my computer.